wiki/Skinning/Interface/en.md

```diff
@@ -464,947 +610,1593 @@ _See also: [Offset Wizard](/wiki/Offset_Wizard)._
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![No][false] | ![Yes][true] | Normal | Centre | 1366x768 |
 
-**Notes:**
+Notes:
 
-- If this is not skinned, the background of the map will be dimmed and used by default.
-- This element is will **not** strech to fit.
+- When the game is paused, the playfield will be dimmed and this file will overlay on top of it.
+- This element is will not strech to fit.
 - Full image height is 768px.
+- Smaller images are shown with transparent borders while larger images are partially shown.
 
 ---
 
-_`fail-background.png`_ (transparent; not included)
+`fail-background.png`
 
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![Yes][true]      |
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![No][false] | ![Yes][true] | Normal | Centre | 1366x768 |
 
-**Notes:**
+Notes:
 
-- If this is not skinned, the background of the map will be dimmed and used by default.
+- When the player has failed, the playfield will be dimmed and this file will overlay on top of it.
 - This element will strech to fit.
 
 ---
 
-![](img/pause-back.png "pause-back.png")
+`pause-back.png`
+
+![](img/pause-back.png)
 
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![Yes][true]      |
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![No][false] | ![Yes][true] | Normal | Centre | - |
 
-**Note:**
+Notes:
 
-- This element appears in the fail and pause screens.
+- This element is positioned at 576px height.
+- This element is seen in the fail and pause screens.
 
 ---
 
-![](img/pause-continue.png "pause-continue.png")
+`pause-continue.png`
 
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![Yes][true]      |
+![](img/pause-continue.png)
 
-- This element appears in the pause screen.
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![No][false] | ![Yes][true] | Normal | Centre | - |
+
+- This element is positioned at 224px height.
+- This element is seen in the pause screen.
 
 ---
 
-![](img/pause-replay.png "pause-replay.png")
+`pause-replay.png`
+
+![](img/pause-replay.png)
 
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![Yes][true]      |
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![No][false] | ![No][false] | Normal | Right | - |
 
-**Notes:**
+Notes:
 
 - This element appears on the ranking screen (after finishing a map or viewing a score).
-- This overrides `ranking-replay.png` in skin versions 2.0+.
+- If skinned, this element overrides `ranking-replay.png` in skin versions 2.0+.
```

Comment:
Remove
>If skinned, this overrides `ranking-replay.png` in skin versions 2.0+.

Same for ranking-retry.
The ranking buttons are skin version 1 exclusive and stating this fact here makes it sound like they are not.
osu! will just use the default skin's buttons if they are not skinned and ignore the ranking buttons.